config/routes.rb

```diff
@@ -1,56 +1,29 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
```

Comment:
Way to delete the tons of boilerplate code that comes in routes.rb. Also good comments on your routes